ci/pre-get_kubeconfig.yml

```diff
@@ -0,0 +1,38 @@
+---
+# This file will be copied to the controller, and then run from there using shell: cmd=`ansible-playbook ...`
+# This is required because I want to use some role from the ci-framework, and the implementation includes some plugins/actions which aren't installed properly. It's because zuul doesn't support installing from collections, and the actions won't exist on the zuul executer.
+- hosts: localhost
+  gather_facts: true
+  #environment:
+  #  ANSIBLE_ROLES_PATH: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].src_dir }}/ci_framework/roles{{ (':' + ansible_env.ANSIBLE_ROLES_PATH ) if ansible_env.ANSIBLE_ROLES_PATH is defined }}"
+  #  ANSIBLE_ACTION_PLUGINS: "{{ ansible_env.HOME}}/{{ zuul.projects['github.com/openstack-k8s-operators/ci-framework'].src_dir }}/ci_framework/plugins/action{{ (':' + ansible_env.ANSIBLE_ACTION_PLUGINS ) if ansible_env.ANSIBLE_ACTION_PLUGINS is defined }}"
+  tasks:
+    - name: Show ansible_env.ANSIBLE_ROLES_PATH
+      debug: var=ansible_env.ANSIBLE_ROLES_PATH
+      when: ansible_env.ANSIBLE_ROLES_PATH is defined
+    - debug: var=ansible_env.ANSIBLE_ACTION_PLUGINS
+      when: ansible_env.ANSIBLE_ACTION_PLUGINS is defined
+
+    #- name: "Log into OpenShift"
+    #  ansible.builtin.shell:
+    #    cmd: |
+    #      oc login -u {{ cifmw_openshift_user }} -p {{ cifmw_openshift_password }} --insecure-skip-tls-verify=true {{ cifmw_openshift_api }}
+
+    #- name: "Link the kubeconfig file"
+    #  ansible.builtin.shell:
+    #    cmd: |
+    #      mkdir -p {{ cifmw_openshift_kubeconfig | dirname }}
+    #      ln -s {{ ansible_env.HOME }}/.kube/config {{ cifmw_openshift_kubeconfig }}
+
+    #  # I need to figure out why the ci_script module is not available in the environment, since the command above is taking the login command out of that.
+    - name: "Login into openshift"
+      ansible.builtin.include_role:
+        name: "openshift_login"
```

Comment:
trying to locate where this role is being defined, is that in https://github.com/rdo-infra/review.rdoproject.org-config?